app/models/wpcc/salary_message.rb

```diff
@@ -0,0 +1,19 @@
+module Wpcc
+  class SalaryMessage
+    include ActiveModel::Model
+    attr_accessor :salary, :salary_frequency, :message
+
+    OPT_IN_SALARY_LOWER_LIMIT = 5_876
+    OPT_IN_SALARY_UPPER_LIMIT = 10_000
+
+    def manually_opt_in_message?
+      message == :manually_opt_in && manually_opt_in?
+    end
```

Comment:
I would do the same as I suggested here: https://github.com/moneyadviceservice/wpcc/pull/73/files#diff-beafab4db73366b96fda2cd86b897d61R15

Because this methods appear to be more on the presenter and the private methods appear to be the business rule of this case.